tests/Query/DbTestCase.php

```diff
@@ -19,13 +21,18 @@ class DbTestCase extends TestCase
 
     public function setUp(): void
     {
-        $this->configuration = new Configuration();
+        if (method_exists(ORMSetup::class, 'createDefaultAnnotationDriver')) {
```

Comment:
We should use attributes if they're available.

```suggestion
        if (PHP_VERSION_ID < 80000) {
```